src/po/fi.po

```diff
@@ -4905,13 +4857,13 @@ msgid "E297: Write error in swap file"
 msgstr "E297: Kirjoitusvirhe swap-tiedostossa"
 
 msgid "E298: Didn't get block nr 0?"
-msgstr "E298: Lohko 0:aa ei saatu?"
```

Comment:
This is not how to inflect numbers written in digits in Finnish, see http://www.kielitoimistonohjepankki.fi/ohje/268